libcontainer/configs/validate/validator.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runc/libcontainer/configs"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```

Comment:
I think it's a explicit or implicit issue, and I'd prefer it to be explicit, we can wait for other maintainers' opinions :)